alot/commands/envelope.py

```diff
@@ -168,6 +168,30 @@ def __init__(self, mail=None, envelope=None, **kwargs):
         self.envelope = envelope
         self.envelope_buffer = None
 
+    def _get_keys_addresses(self):
+        addresses = set()
+        for key in self.envelope.encrypt_keys.values():
+            for uid in key.uids:
+                addresses.add(uid.email)
+        return addresses
+
+    def _get_recipients_addresses(self):
+        addresses = set()
+        for header in ['To', 'Cc']:
+            for value in self.envelope.headers.get(header) or []:
+                for recipient in value.split(','):
+                    if recipient:  # may be an empty string
+                        _, address = email.utils.parseaddr(recipient)
+                        addresses.add(address)
+        return addresses
+
+    def _is_encrypted_to_all_recipients(self):
+        recipients = self._get_recipients_addresses()
+        keys = self._get_keys_addresses()
+        if not recipients.issubset(keys):
+            return False
```

Comment:
`return recipients.issubset(keys)` ?
Also, perhaps better call `keys` `addresses_fom_keys` or similar to not cunfuse the value with actual gpg-keys..